nutils/transform.py

```diff
@@ -85,8 +86,9 @@ class TransformItem(types.Singleton):
 
     __slots__ = 'todims', 'fromdims'
 
-    @types.apply_annotations
-    def __init__(self, todims, fromdims: int):
+    def __init__(self, todims: Integral, fromdims: Integral):
```

Comment:
Why `Integral` and not `int`?

Reply:
By using `Integral` we allow `numpy.int_` in addition to `int`, which would otherwise require hunting down and preprocessing all arguments that originate from a numpy operation. The hash of both types corresponds so for `Singleton`'s purpose it is fine to take values as they come.

Reply:
Ok.